scheduler/src/cook/components.clj

```diff
@@ -291,6 +292,13 @@
                                   offer-incubate-ms)
      :mesos-master (fnk [[:config [:mesos master]]]
                         master)
+
+     :mesos-master-hosts (fnk [[:config [:mesos master {master-hosts nil}]]]
+                              (if master-hosts
```

Comment:
The important thing with the diagnostic messages on the main input is to
not use Schema, but instead try to provide actionable instructions to the
user.

On Fri, Nov 20, 2015 at 3:26 PM Li Jin <notifications@github.com> wrote:

> In scheduler/src/cook/components.clj
> <https://github.com/twosigma/Cook/pull/85#discussion_r45514227>:
>
> > @@ -291,6 +292,13 @@
> >                                    offer-incubate-ms)
> >       :mesos-master (fnk [[:config [:mesos master]]]
> >                          master)
> > +
> > +     :mesos-master-hosts (fnk [[:config [:mesos master {master-hosts nil}]]]
> > +                              (if master-hosts
>
> Hmm...Wonder if we can do schema checks instead of checking this manually.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/twosigma/Cook/pull/85/files#r45514227>.
>